modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -101,13 +105,40 @@ ocis.yaml
 
 When using `ocis.yaml` and you configure a service, the topic for the service configuration must be the service name.
 
-You can list the possible services names by typing:
+You can list the available services names by typing:
 
 [source,bash]
 ----
 ocis list
 ----
 
+=== Special Configuration Settings
+
+Special configuration settings can be the following:
+
+* Configurations that can only be made with yaml but not with environment variables.
+* yaml configurations that use an OS environment variable for the value.
+
+.yaml only settings
+There are certain situations where multiple settings using the same keys apply for a configuration set. This can be like in the xref:{s-path}/app-registry.adoc[app-registry] service where default apps for mimetypes are defined. A single environment variable cannot hold all that information. For this case, a yaml configuration is the only way possible. Note that both installation methods, binary and container, can deal with yaml configuration files.
+
+.Using OS environment variables in yaml files
+OS environment variables can be used in yaml config files for Infinite Scale services which will be replaced by the actual value of the environment variable at runtime. This method allows to define a default setting, but use different values for different use cases.
```

Comment:
```suggestion
OS environment variables can be used in yaml config files for Infinite Scale services which will be replaced by the actual value of the environment variable at runtime. This method allows a default setting to be defined, but use different values for different use cases.
```

Reply:
How would someone specify a default setting?
In the example below, every individual installation instance has to define `SOME_HTTP_ADDR` - there is no "default" specified.

Can we give an example where a default value is specified, along with an env var that overrides the default, if set?